src/Sets/PolynomialZonotope.jl

```diff
@@ -6,7 +6,10 @@ export PolynomialZonotope, dim, σ,
        minkowski_sum
 
 """
-    PolynomialZonotope{N}
+    PolynomialZonotope{N, VTC<:AbstractVector{N},
```

Comment:
my preferred choice would be:

```
struct PolynomialZonotope{N, VT<:AbstractVector{N}, MT<:AbstractMatrix{N}}
... # use VT and MT
end
```
because you can reason about polynomial zonotopes with regular / sparse / static arrays , = dispatch on `VT` and `MT`. somtimes it is convenient to use array views and it's pertinent to generalize the abstract vector container as well.

Reply:
> somtimes it is convenient to use array views and it's pertinent to generalize the abstract vector container as well.

I also started with removing the type restrictions completely but the problem is that we somehow need to bind `N` and for that we need a container type.

Reply:
If a struct can have unused type parameters (not tested), we could determine the `N` parameter in the constructor via `typeof(center[1])`. Should I try?

Reply:
I did that now. What do you think?